unfoldingWord/translationCore
# TC Wifi Fix On Hibernate

#### This pull request addresses:
This PR fixes the issues related to going offline after the computer is hibernate mode.


#### How to test this pull request:
To test this you can have your wifi on then close you computer wait about 20 mins open back up, and see if your wifi is still on.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/914)
